Documentation/CodingStyle.md

```diff
@@ -496,24 +496,31 @@ UniqueObject& my_unique_object(); // Free function.
 
 ### Comments
 
-Make comments look like sentences by starting with a capital letter and ending with a period (punctuation). One exception may be end of line comments like this `if (x == y) // false for NaN`.
+Comments should be written using `//` and not `/* */`, except for the copyright notice.
+
+Make comments look like sentences by starting with a capital letter and ending with a period (punctuation). One exception may be end of line comments like this: `if (x == y) // false for NaN`
```

Comment:
Maybe add an exception for verbatim spec comments?

Reply:
Ah good point!